api.swagger.yaml

```diff
@@ -12263,7 +12256,34 @@ components:
               type: boolean
               example: false
               readOnly: true
+            free_skills:
+              description: Free Skills
+              type: array
+              items:
+                type: string
+            interests:
```

Comment:
This is also a taxonomy, but we dont have any EPs for them yet (they are already indexed in solr).
should be an object array
should be called personal_interests.
NOTE: the UI is misleading, looks like a free string, but its comma separated taxonomies under the hood.